src/nprintml/net/bootstrap.py

```diff
@@ -0,0 +1,266 @@
+"""nPrint installation bootstrapping command"""
```

Comment:
I went back and forth as to where this module should live, but ultimately determined it can and should be stowed away with "stuff to do with networking / nPrint," in this subpackage `nprintml.net`.

Forthcoming work to invoke nPrint can be added here as well.